test/integration-test-v2.py

```diff
@@ -259,15 +256,29 @@ def test_revoke_by_privkey():
         cleanup()
 
     # Create a new client with the JWK as the cert private key
```

Comment:
The old code here was buggy, since new_client was never used. However, now that that's deleted it becomes clear that this test has been using the wrong client all along, and so hasn't been testing revocation by privkey; it's been testing revocation by possession of the issuing account. We should probably delete the whole test rather than leave it around in a misleading state.

Reply:
> We should probably delete the whole test rather than leave it around in a misleading state.

Can do :+1: